config.go

```diff
@@ -19,9 +20,10 @@ import (
 )
 
 const (
-	AwsCloud   = "aws"
-	GcpCloud   = "gcp"
-	AzureCloud = "azure"
+	AwsCloud            = "aws"
+	GcpCloud            = "gcp"
+	AzureCloud          = "azure"
+	LoadBalancerCommand = "kubectl get svc ingress-nginx-ingress-controller -o jsonpath='{.status.loadBalancer.ingress[0].hostname}'"
```

Comment:
This should be in a different const block. This is sort of an enum type for cloud